migrations/sql/V14__Index_file_path_on_file_table.sql

```diff
@@ -0,0 +1,5 @@
+/**
```

Comment:
There's already a migration 14, so you should rebase this branch onto the latest master.

Reply:
Resolved